index.html

```diff
@@ -1388,6 +1390,64 @@ <h3>Designing character escapes</h3>
 	<p><a href="https://www.w3.org/TR/charmod/#sec-Escaping">Character Escaping, C042</a>, in <cite>Character Model for the World Wide Web: Fundamentals</cite></p>
 	</details>
 	</div>
+	
+	<div class="example" id="char-escape-examples">
+		<p>Here are some examples of common escaping mechanisms found on the Web or in common programming languages. The example character here is &#x1f63d; <code class=uname translate=no>U+1F63D</code>.</p>
+		<table>
+			<thead>
+				<tr>
+					<th width="20%">Type</th>
+					<th>Application</th>
+					<th width="20%">Example</th>
+					<th>Description</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>Hex NCRs</td>
```

Comment:
```suggestion
					<td>Hex <abbr title="numeric character references">NCRs</abbr></td>
```

Reply:
Good callouts. See if my edits in ff085fe address these.